h2/events.py

```diff
@@ -136,6 +136,51 @@ def __repr__(self):
         )
 
 
+class ResponseSent(object):
```

Comment:
So I think we'd like to clarify in some way that these are internal events. It's probably enough to put appropriate docstrings on them. Otherwise this is fine.

Out of interest, why don't we need RequestSent?